Source/Android/app/src/main/res/values/strings.xml

```diff
@@ -229,7 +229,11 @@
     <string name="aspect_ratio">Aspect Ratio</string>
     <string name="aspect_ratio_description">Select what aspect ratio to use when rendering</string>
     <string name="shader_compilation_mode">Shader Compilation Mode</string>
-    <string name="shader_compilation_mode_description">Specifies when to use Ubershaders. Disabled - Never, Hybrid - Use ubershaders while compiling specialized shaders. Exclusive - Use only ubershaders, largest performance impact. Skip Drawing - Do not draw objects while shaders are compiling, will cause broken effects.</string>
+    <string name="shader_compilation_mode_description">Specifies when to use Ubershaders.</string>
+    <string name="shader_compilation_sync_description">Synchronous: Ubershaders are never used. Stuttering will occur during shader compilation, but GPU demands are low. Recommended for low-end hardware.\nIf unsure, select this mode.</string>
+    <string name="shader_compilation_sync_uber_description">Synchronous (Ubershaders): Ubershaders will always be used. Provides a near stutter-free experience at the cost of high GPU performance requirements. Only recommended for high-end systems.</string>
+    <string name="shader_compilation_async_uber_description">Asynchronous (Ubershaders): Ubershaders will be used to prevent stuttering during shader compilation, but specialized shaders will be used when they will not cause stuttering. In the best case it eliminates shader compilation stuttering while having minimal performance impact, but results depend on video driver behavior.</string>
+    <string name="shader_compilation_async_skip_description">Asynchronous (Skip Drawing): Prevents shader compilation stuttering by not rendering waiting objects. Can work in scenarios where Ubershaders doesn\'t, at the cost of introducing visual glitches and broken effects. Not recommended, only use if the other options give poor results on your system.</string>
```

Comment:
<pre>doesn\'t</pre>

I'm assuming that will be "doesn't" on devices, right?

Reply:
Looks good on my device and apostrophes are done the same way elsewhere in strings.xml. 
![Screenshot_20190320-200548_Dolphin Emulator](https://user-images.githubusercontent.com/17330088/54726976-c6b6a600-4b4b-11e9-8576-361784c6e920.jpg)